### PR TITLE
fix for aarch64-apple-ios

### PR DESCRIPTION
**Pull request by Khronos31**
This is not your bug. I have made an interim fix.
https://github.com/rust-lang/rust-bindgen/issues/1211

### CODE REVIEW

**Comment by simlay** [comment]
Full disclosure, I know nothing about onig but I've done a bunch on [iOS/objective-c support for bindgen](https://github.com/rust-lang/rust-bindgen/pulls?q=is%3Apr+author%3Asimlay) in the last year. This is kinda the way I'd suggest doing it until rust-lang/rust-bindgen#1211 has a resolution. It's unclear if the new apple laptops which are aarch64 will require similar work.

File: onig_sys/build.rs [hunk 1]
```diff
@@ -189,10 +189,18 @@ fn compile() {
...
     #[cfg(feature = "generate")]
     {
+        let arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap();
+        let os = env::var("CARGO_CFG_TARGET_OS").unwrap();
         let bindings = bindgen::Builder::default()
             .header(_path)
             .derive_eq(true)
             .layout_tests(false)
+            .clang_arg(
+            if arch == "aarch64" && os == "ios" {
```
Comment (iwillspeak) [inline onig_sys/build.rs:199]:
```suggestion
            // Work around for #rust-Lang/rust-bindgen/1211
            if arch == "aarch64" && os == "ios" {
```

Comment (simlay) [reply onig_sys/build.rs:199]: Looks like that issue got resolved.

Comment (iwillspeak) [reply onig_sys/build.rs:199]: Do we need this PR any more? #147 Updates to bindgen .55. Hopefully that contains the fix.

**Comment by iwillspeak** [comment]
Closing this as I hope we don't need it any more with `bindgen` 0.55. If it turns out we _do_ still need this change I'm happy to get it merged if someone shouts up on this PR.

**Pull request closed without merging**